Doc/tutorial/controlflow.rst

```diff
@@ -519,6 +519,169 @@ and of course it would print:
 Note that the order in which the keyword arguments are printed is guaranteed
 to match the order in which they were provided in the function call.
 
+Special parameters
+------------------
+
+By default, arguments may be passed to a Python function either by position
+or explicitly by keyword. For readability and performance, it makes sense to
+restrict the way arguments can be passed so that a developer need only look
+at the function definition to determine if items are passed by position, by
+position or keyword, or by keyword.
+
+A function definition may look like::
+
+   def f(pos1, pos2, /, pos_or_kwd, *, kwd1, kwd2):
+         -----------    ----------     ----------
```

Comment:
Distinct lack of [Box-drawing characters](https://en.wikipedia.org/wiki/Box-drawing_character), tut tut!

(For those who don't know that Pablo and I are friends, this is gentle ribbing, not me being a jerk 😉)

Reply:
Sphinx does something fancy with those characters and breaks the whole diagram :(

Reply:
@pganssle Do you know a code block in sphinx that does not break box-drawing characters?

Reply:
I don't see anything fancy, am I missing something?

![box-chars](https://user-images.githubusercontent.com/1377457/57584240-d517a300-74a6-11e9-9419-80eb3366c516.png)

Looks like they work just fine.

Reply:
The problem happens when you change all of the characters. Try it out and meanwhile I will try to add a screenshot.

Reply:
![box-chars](https://user-images.githubusercontent.com/1377457/57586793-0ef9a100-74c9-11e9-9a1d-5f5199e6ae59.png)

Still looks fine to me with all box characters.

Reply:
Can you copy here the text? I may be doing something wrong....

Reply:
```
.. code-block:: none

   def f(pos1, pos2, /, pos_or_kwd, *, kwd1, kwd2):
         ──┬───────     ─┬────────     ─────┬────
           │             │                  │
           │        Positional or keyword   │
           │                                └ Keyword only
           └── Positional only
```

This is what I have.

Reply:
This is how it renders for me (copy-pasting you code):

![Screenshot from 2019-05-12 20-53-00](https://user-images.githubusercontent.com/11718525/57587140-fce32700-74f7-11e9-9f46-4066bc5e36b8.png)

Notice that in the editor it renders nicely:

![Screenshot from 2019-05-12 20-53-46](https://user-images.githubusercontent.com/11718525/57587143-16846e80-74f8-11e9-8e35-015021b81342.png)

Reply:
Is this a problem with the font you are using in the browser? I tried this with two different browsers and it's rendering correctly.

Reply:
I changed the font a couple of times and I have the same problem :S

Reply:
On the other hand, firefox does it correctly. Can you try with Chrome?

Reply:
@pablogsal It works fine on Chrome for me. Maybe we should leave it as is and open a new BPO issue about box characters not working correctly in documentation?

Seems like something that can probably be fixed with a webfont or something, but no need to hold up this PR on fixing it.